spec/models/category_spec.rb

```diff
@@ -0,0 +1,5 @@
+require 'rails_helper'
+
+RSpec.describe Category, type: :model do
```

Comment:
if you manually create your files, you wont get all these extras. would like to see validation tests here.